src/ipLoadBalancing/farm/ipLoadBalancing-farm-server.lexi.service.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+_.forEach(["tcp", "udp", "http"], function (type) {
+    angular
+        .module("ovh-api-services")
+        .service("OvhApiIpLoadBalancingFarm" + _.capitalize(type) + "ServerLexi",
+                 ["$resource", "$cacheFactory", function ($resource, $cacheFactory) {
```

Comment:
No need to prefix your function with injection, `ngAnnonate` do this for us :)

Reply:
Not in this case, unfortunately...